docs/docs-content/integrations/aws-autoscaler.md

```diff
@@ -26,393 +29,781 @@ conditions:
+| `charts.cluster-autoscaler.cloudProvider`             | The cloud provider where the autoscaler will be deployed.                                                                                                                                                                                                                                                   | `aws`                                | Yes          |
+| `charts.cluster-autoscaler.extraArgs.expander`        | Indicates which Auto Scaling Group (ASG) to expand. Options are `random`, `most-pods`, and `least-waste`. `random` scales up a random ASG. `most-pods` scales up the ASG that will schedule the most amount of pods. `least-waste` scales up the ASG that will waste the least amount of CPU/MEM resources. | `least-waste`                        | Yes          |
+
+### Usage
+
+The Helm-based Cluster Autoscaler pack is available for Amazon EKS host clusters. To deploy the pack, you must first
+define an IAM policy in the AWS account associated with Palette. This policy allows the Cluster Autoscaler to scale the
+cluster's node groups.
+
+Use the following steps to create the IAM policy and deploy the Cluster Autoscaler pack.
+
+1. In AWS, create a new IAM policy using the snippet below and give it a name, for example,
+   _PaletteEKSClusterAutoscaler_. Refer to the
+   [Creating IAM policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) guide for
+   instructions.
+
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Action": [
+           "autoscaling:DescribeAutoScalingGroups",
+           "autoscaling:DescribeAutoScalingInstances",
+           "autoscaling:DescribeLaunchConfigurations",
+           "autoscaling:DescribeScalingActivities",
+           "autoscaling:DescribeTags",
+           "ec2:DescribeInstanceTypes",
+           "ec2:DescribeLaunchTemplateVersions"
+         ],
+         "Resource": ["*"]
+       },
+       {
+         "Effect": "Allow",
+         "Action": [
+           "autoscaling:SetDesiredCapacity",
+           "autoscaling:TerminateInstanceInAutoScalingGroup",
+           "ec2:DescribeImages",
+           "ec2:GetInstanceTypesFromInstanceRequirements",
+           "eks:DescribeNodegroup"
+         ],
+         "Resource": ["*"]
+       }
+     ]
+   }
+   ```
+
+2. Copy the IAM policy
+   [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html). Your policy ARN
+   should be similar to `arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler`.
```

Comment:
```suggestion
   should be similar to `arn:aws:iam:: 123456789:policy/PaletteEKSClusterAutoscaler`.
```

Eh 😅, let's avoid putting our account number in public. It's not really aa sensitive value but let's make malicious actor's life harder

Reply:
Great catch! :sweat_smile: